### PR TITLE
remove own mode

### PR DESCRIPTION
**Pull request by StarfallProjects**

### CODE REVIEW

File: docs/release-notes.md
Comment (Joffcom) [inline docs/release-notes.md:80]: We have removed `executions_process` fully so `main` isn't an option anymore either, The remaining setting is `execitons_mode` which is `regular` or `queue`

**Review by Joffcom — APPROVED** [review]

**Comment by StarfallProjects** [comment]
Converting to draft to ensure we don't merge until 1.30.0
Need to update release note before merging

**Review by despairblue — APPROVED** [review]
Looks good. I could not find any other mentions of own mode.

**Review by despairblue — APPROVED** [review]
Looks good. I could not find any other mentions of own mode.

**Pull request merged**
